crates/pipeline_manager/src/integration_test.rs

```diff
@@ -1209,6 +1209,95 @@ async fn primary_keys() {
         .await;
 }
 
+/// Test case-sensitive table ingress/egress behavior.
+#[actix_web::test]
+#[serial]
+async fn case_sensitive_tables() {
+    let config = setup().await;
+    // Table "TaBle1" and view "V1" are case-sensitive and can only be accessed
+    // by quoting their name.
+    // Table "v1" is also case-sensitive, but since its name is lowercase, it
+    // can be accessed as both "v1" and "\"v1\""
+    let _ = deploy_pipeline_without_connectors(
+        &config,
+        r#"create table "TaBle1"(id bigint not null);
+create table table1(id bigint);
+create view "V1" as select * from "TaBle1";
```

Comment:
Maybe the compiler should give a warning about this case

Reply:
This is supposed to be completely valid SQL.

Reply:
Not every program you can write in a language is a good program.